hosting/src/models/event.ts

```diff
@@ -0,0 +1,32 @@
+import dayjs, { Dayjs } from 'dayjs'
+
+import { Event } from '@phits-tech/common/dist/dao-firestore/model-types'
+
+export interface EventUi {
+  slug: string
+  name: string
+  description: string
+  dateStart: Dayjs
+  dateEnd: Dayjs
+  time: string
+  location: string
+  series?: string
+  seriesType?: string
+  badges?: string[]
+  hostId?: string
+  hostName?: string
+  registrationRequired?: boolean
+  registrationUrl?: string
+}
+
+export function eventToEventUi(event: Event): EventUi {
+  const dsDayjs = dayjs.unix(event.dateStart.seconds)
+  const deDayJs = dayjs.unix(event.dateEnd.seconds)
+
+  return {
+    ...event,
+    dateStart: dsDayjs,
+    dateEnd: deDayJs,
+    time: dsDayjs.format('ddd DD MMM, HH:mm') + '-' + deDayJs.format('HH:mm')
```

Comment:
I think we should remove `time` from this. It's very context sensitive. Can we do it directly in `<template>`?

Reply:
I tried doing this directly in the template and hated it. It looks terrible. And it appears we do actually reuse it.

We should probably extract it as a function. [Filters were removed in Vue 3](https://v3.vuejs.org/guide/migration/filters.html#overview); they suggest registering `$filters` under `globalProperties`.

Reply:
DONE